packages/trader/src/AppV2/Containers/Trade/trade-desktop.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import clsx from 'clsx';
+import { observer } from 'mobx-react';
+import { useStore } from '@deriv/stores';
+import { Loading } from '@deriv/components';
+import { useLocalStorageData } from '@deriv/hooks';
+import { useTraderStore } from 'Stores/useTraderStores';
+import PurchaseButton from 'AppV2/Components/PurchaseButton';
+import { getChartHeight } from 'AppV2/Utils/layout-utils';
+import { TradeParametersContainer, TradeParameters } from 'AppV2/Components/TradeParameters';
+import CurrentSpot from 'AppV2/Components/CurrentSpot';
+import { TradeChart } from '../Chart';
+import { isDigitTradeType } from 'Modules/Trading/Helpers/digits';
+import TradeTypes from './trade-types';
+import MarketSelector from 'AppV2/Components/MarketSelector';
+import useContractsForCompany from 'AppV2/Hooks/useContractsForCompany';
+import AccumulatorStats from 'AppV2/Components/AccumulatorStats';
+import OnboardingGuide from 'AppV2/Components/OnboardingGuide/GuideForPages';
+import { sendSelectedTradeTypeToAnalytics } from '../../../Analytics';
+
+const TradeDesktop = observer(() => {
+    const chart_ref = React.useRef<HTMLDivElement>(null);
+    const {
+        client: { is_logged_in },
+        ui: { is_dark_mode_on },
+    } = useStore();
+    const {
+        active_symbols,
+        contract_type,
+        has_cancellation,
+        is_accumulator,
+        is_market_closed,
+        onChange,
+        onMount,
+        onUnmount,
+        symbol,
+    } = useTraderStore();
+    const { trade_types } = useContractsForCompany();
+    const [guide_dtrader_v2] = useLocalStorageData<Record<string, boolean>>('guide_dtrader_v2', {
+        trade_types_selection: false,
+        trade_page: false,
+        positions_page: false,
+    });
+
+    const symbols = React.useMemo(
```

Comment:
symbols and onTradeTypeSelected same line of code repeated twice. IMO let it be as it is for now. But once we start working on desktop and the logic still stays here then we have to refactor it.